### ISSUE TITLE
Blacklist to ignore specific folders and files

### ISSUE THREAD

**Issue by demlak** [FabianSperrle/kirby-stats#18]
Hi, the stats are flooded from various files and folders, that i don't want to see stats of. Maybe there is a solution to filter them?

thx a lot

**Comment by BlackStonedLover** [comment]
Yeah would be a greate feature cause atm all images get also tracked. which kind of sucks.
